---
layout: default
title: Methods Examples
author: Herrington, Kari
date: Feb 11, 2025
---

## Brightfield Imaging with Large imaging

[Brightfield images were captured on an inverted Nikon TiE microscope
(Nikon) run using NIS-Elements 5.21.03 (b1489_64bit) that was equipped
with a fully automated Nikon stage, the Nikon Perfect Focus System,
halogen illumination housing, and a 2.5X C-mount adaptor in front of
the DS-Ri2 Color Camera (Nikon). Images were captured under Kohler
illumination, using a Plan Apo 40x/ 0.95 NA Corr Objective (Nikon),
pixel size 220nm, 30ms exposure, software white balance (red 1.07, Blue
3.12), Gain at 1.4x, a captured shading correction applied during
imaging, and tile scanned using HCA/jobs to set coordinates with a 15%
overlap, then stitched with
NIS-elements. ]

\

## [CSU-W1 (before laser change in Feb 2024)]

[Excerpt from ]

Calia, Giuliana P., Xinyue Chen, Binyamin Zuckerman, and Leor S.
Weinberger. "Comparative Analysis between Single-Cell RNA-Seq and
Single-Molecule RNA FISH Indicates That the Pyrimidine Nucleobase
Idoxuridine (IdU) Globally Amplifies Transcriptional Noise." *bioRxiv*,
March 15, 2023, 2023.03.14.532632.
[https://doi.org/10.1101/2023.03.14.532632](https://doi.org/10.1101/2023.03.14.532632).

\

Images were taken on an inverted Nikon TiE microscope (Nikon) run using
Micromanager 2.0 \[NEEDS TO BE CITED\] equipped with a CSU-W1 Spinning
Disk with Borealis Upgrade (Yokogawa, Andor), ILE Laser launch with 4
laser lines (450/488/561/646nm, Andor), quad-band dichroic
ZT405/488/561/647 (Chroma),  emission filters for DAPI (ET447/60), GFP
(ET525/50), RFP (ET607/36), and Cy5 (ET685/40) (Chroma),  piezo XYZ
stage (ASI), and Zyla 4.2 CMOS camera (Andor),  using a Plan Apo VC
60x/1.4 Oil objective (Nikon). Approximately 10 XY locations were
randomly selected for each condition. For each image, XY pixel size was
108nm/px, and a Z-step size of 250nm was used with over 60 image planes
to fully cover the tissue.

\

MicroManager Citations

Mark A Tsuchida, Nenad Amodaj, Henry Pinkard, Ronald D Vale, and Nico
Stuurman (2014), Advanced methods of microscope control using μManager
software. **Journal of Biological Methods** 2014
1(2):e11 [\<doi:10.14440/jbm.2014.36\>](http://dx.doi.org/10.14440/jbm.2014.36)

Arthur Edelstein, Nenad Amodaj, Karl Hoover, Ron Vale, and Nico Stuurman
(2010), Computer Control of Microscopes Using μManager. **Current
Protocols in Molecular Biology** 14.20.1-14.20.17

Page on Citing Micromanager:
[https://micro-manager.org/Citing_Micro-Manager](https://micro-manager.org/Citing_Micro-Manager)

\

## [CSU-W1 (before laser change in Feb 2024)]

[Excerpt from:]

Touhara, Kouki K., Nathan D. Rossen, Fei Deng, Tifany Chu, Andrea M.
Harrington, Sonia Garcia Caraballo, Mariana Brizuela, et al. "Crypt and
Villus Enterochromaffin Cells Are Distinct Stress Sensors in the Gut."
bioRxiv, April 25, 2024.
[https://doi.org/10.1101/2024.02.06.579180](https://doi.org/10.1101/2024.02.06.579180).

\

Confocal images were captured on an inverted Nikon Ti microscope run
using Micro Manager 2.0 Gamma
(*[54](https://www.biorxiv.org/content/10.1101/2024.02.06.579180v2.full#ref-54)*), equipped with a Zyla 4.2 CMOS camera (Andor), piezo
XYZ stage (ASI), CSU-W1 Spinning Disk with Borealis upgrade
(Yokogowa/Andor), Spectra-X (Lumencor), ILE 4 line Laser Launch
(405/488/561/640 nm; Andor). Images were taken using a Plan Apo λ 20x /
0.75 using lasers 405, 488, and 561 nm and emission filters 447/60,
525/50, 607/36, for DAPI, GFP, and RFP, respectively. Maximum intensity
projections were generated in Fiji v2.14.

\

## Crest-V2 spinning disk

Excerpt from

Zhong, Xiaofang, Qiongyu Li, Benjamin J Polacco, Trupti Patil, Aaron
Marley, Helene Foussard, Prachi Khare, et al. "A Proximity Proteomics
Pipeline with Improved Reproducibility and Throughput." *Molecular
Systems Biology* 20, no. 8 (August 2, 2024): 952--71.
[https://doi.org/10.1038/s44320-024-00049-2](https://doi.org/10.1038/s44320-024-00049-2).

\

[Images were generated using a Nikon Ti2-E microscope equipped with a
Crest X-Light-V2 spinning disk confocal (Crest Optics), emission
filters, 438/24 (DAPI), 511/20 (GFP), 560/25 (RFP), and 685/40 (Cy5),
and Celeste Light Engine excitation lasers 405/477/546/638 nm used
respectively (Lumencor), Piezo stage (Mad City Labs), and a Prime 95B
25 mm CMOS camera (Photometrics) using a Plan Apo VC 100x/1.4 Oil
(Nikon). The data was captured with NIS-Elements software (v. 5.41.01
build 1709, Nikon) and processed in Fiji/ImageJ2 (Schindelin et
al, ][2012](https://www.embopress.org/doi/full/10.1038/s44320-024-00049-2#core-CR62)[).]

## Crest-V2 Spinning Disk with FRAP

Adapted after a preprint

[ Condensates were allowed to settle in the well for at least 1 hour
before imaging with a [Plan Apo [VC
[100x[/1.4NA[
[(Nikon)[
[oil immersion objective on
a[ Nikon Ti2 microscope equipped
with a[ CREST[-V2[ LFOV
Spinning Disk[ (Crest Optics),
Celest[a[ Light Engine
([Lumencor[)[, [with
excitati[on wave lengths[
5[46nm [or 647
[and emission filters
[FF01-595/31[ or
FF02-685/40[ ([Semrock[)[ [to Image
Alexa Fluor [555 or 647
respectively,[ [a 405/488/561/640/750
Dichroic (Crest)[. [a[ Prime 95B
[25mm[ [sCMOS[ camera[, run using
NIS Elements [5.41.01 build 1709
([Nikon)[ [ [.[ Fluorescence recovery after
photobleaching was achieved with [a Opti[-[Microscan[ FRAP unit[
[wi[th [a
[ [405[nm Reflect/430-800nm Trans Dual
TIRF (FRAP) [dichroic[ and
[405nm laser for photobleaching.
20% laser power was used to bleach Alexa Fluor 555-labeled H1 and 15%
laser power was used to bleac[h[ A[lexa Fluor 647-labeled chromatin
arrays.

\

## [CVRI Spinning disk

[Adapted after preprint

[[[Confocal images were taken using a[[[n inverted[[ Nikon[[[ Ti microscope equipped with a
CSU-X1(Yokogawa)[[ spinning disk confocal
[[[using[[[ [[a [[[Plan Apo VC [[60 × 1.4[[[NA (Nikon)[[, oil
objective[[[, 4 line[[[ [[[Vortan[[[
[[[laser launch
(405/488/561[[[[/647nm[[[[)[[[[, with respective emission
filters[[[[, E[[[[T460/50m,
ET525/50m[[[[, FF01-609/57,
[[[[ET630/75m[[[[ (Chroma[[[[, [[[[Semrock[[[[), [[[[am ASI [[[[XY stage with Z piezo,
[[[[and a Prime [[[[95 [[[[s[[[[CMOS[[[[ camera
(Photometrics)[[[.

\

## [[[[CSU-22

[[[[Written for Fixed slides

[[[[A fully automated Nikon Ti equipped with a CSU-22 spinning disk
confocal (Yokogawa), piezo stage (Mad City Labs)  4-line Coherent OBIS
laser launch 405/488/561/640nm (Solamere), a quad dichroic
405/491/561/640 (Yokogawa), and corresponding emission filters
ET460/50m, ET525/50m, ET610/60m, ET700/75m in a filter wheel controlled
by a Lambda 10-3B (Sutter)  for channels DAPI/GFP/RFP/Cy5 respectively.
Images were captured using a Plan Apo VC 60x/1.4 Oil (Nikon) and a 1.7x
relay lens in front of the Photometrics Evolve Delta EMCCD Camera
(154nm/pixel).

\

OMX

Here was text used by Rose for methods
([https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190530#sec003](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190530#sec003))

### Microscopy acquisition and analysis

SIM data was acquired on a DeltaVision [OMX] SR microscope (GE Healthcare) using a 60x/1.42 NA oil immersion
PSF objective and three sCMOS cameras. Immersion oil with a refractive
index of 1.514 was used to match with the refractive index of the
sample. Z stacks of 9μm thickness were collected with 0.125μm step-sizes
using 5 phases and 3 angles per image. Reconstructions were preformed
using SoftWorx 6.5.2 (GE Healthcare) with a 0.003 Wiener filter for the
Alexa Fluor 488 channel and 0.002 Wiener filter for Alexa Fluor 555 as
suggested by analysis using the ImageJ plugin SIMcheck
\[[22](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190530#pone.0190530.ref022)\]. Channel specific
optical transfer functions were used for
reconstructions. [Alignment], averaging and
calculation of the radial distribution were performed using custom
python scripts previously described
\[[5](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0190530#pone.0190530.ref005)\]. Monomeric CM2 data
was fit with an additional central Guassian to distinguish the central
intensity from the ring distribution.

\

\

\
